src/bin/tikv-ctl.rs

```diff
@@ -1217,9 +1217,7 @@ fn from_hex(key: &str) -> Vec<u8> {
     const HEX_PREFIX: &str = "0x";
     let mut s = String::from(key);
     if s.starts_with(HEX_PREFIX) {
-        let len = s.len();
-        let new_len = len.saturating_sub(HEX_PREFIX.len());
-        s.truncate(new_len);
+        s.drain(..HEX_PREFIX.len());
```

Comment:
please add a test for it.

Reply:
I think you can use `key.trim_left_matches()`.